api/src/pcapi/core/users/external/zendesk_sell.py

```diff
@@ -512,13 +512,17 @@ def update_venue(venue: offerers_models.Venue) -> None:
     if venue.isVirtual or _stub_in_test_env("update", venue):
         return
 
+    logger.info("update_venue: venue.id=%s", venue.id)
+
     # API calls to Zendesk Sell are delayed in a GCP task to return quickly
     zendesk_sell_tasks.update_venue_task.delay(zendesk_sell_tasks.VenuePayload(venue_id=venue.id))
 
 
 def do_update_venue(venue_id: int) -> None:
     """Called asynchronously by GCP task"""
+    logger.info("do_update_venue: venue_id=%s", venue_id)
     venue = offerers_repository.find_venue_by_id(venue_id)
+    logger.info("do_update_venue: venue=%s", venue)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** 
```
    logger.info("before do_update_venue: venue_id=%s", venue_id)
    logger.info("after do_update_venue: venue_id=%s", venue_id)
```

Pour dissocier les logs ? 
‏

Reply:
Je ne comprends pas trop le before/after que tu suggère, ici j'ai un log `venue_id` et un log `venue`.